.github/workflows/snyk.yaml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   security:
     runs-on: ubuntu-latest
+    if: (github.repository_owner == 'openfga' && (github.actor != 'dependabot[bot]' && github.actor != 'snyk-bot'))
```

Comment:
Should we run snyk and semgrep even on dependabot PRs? Why did we ignore these actor(s) in the first place?

Reply:
At the moment the secrets are not set for dependabot except for FOSSA, we'd need to renable once the secrets are added